src/routes_initializer.coffee

```diff
@@ -17,12 +17,14 @@ class RoutesInitializer
 
   constructor: (@routesFile, @reverseRoutes, @addRoute) ->
     @_routes = []
+    @_scope = ''
     require(@routesFile) @getRouteDirectives()
 
 
   # Returns a route directive for a specific http method to be called in a routes file
   getHttpDirective: (httpMethod) ->
     (url, {to, as}) =>
+      url = "#{@_scope}#{url}"
```

Comment:
Isn't that dependent on the file system? What is this ran on windows? You can force it to use the posix form. 
```
url = path.posix.join @_scope, url
```
I don't really like that and would suggest just adding a "/" to scope if it doesn't end with one